pages/index.vue

```diff
@@ -3,7 +3,11 @@ import VCard from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VCar
 import VByline from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VByline.vue'
 import useImageUrl from '~~/composables/useImageUrl'
 
-const featuredArticles = await findFeaturedArticlePages('').then(({ data }) =>
+const featuredArticle = await findPage('/').then(({ data }) =>
+  normalizeHomePageFeaturedArticle(data)
```

Comment:
for example:
```
const featuredArticle = value.pageCollectionRelationship?.[0].pages?.[0]
return featuredArticle && normalizeArticle(featuredArticle)
```

Reply:
Gotcha - I did

`const featuredArticle = await findPage('/').then(({ data }) =>
  normalizeArticlePage(data.value.pageCollectionRelationship?.[0].pages?.[0])
)`

I thought you wanted a new function to handle this, but you actually said you just wanted a new type to handle the home.HomePage - but I ended up not needing this